ui/imports/shared/panels/AnimatedText.qml

```diff
@@ -7,7 +7,7 @@ SequentialAnimation {
 
     property var target: null
     property color fromColor: Theme.palette.directColor1
-    property color toColor: Theme.palette.baseColor5
```

Comment:
@saledjenic updated this when resolving the merge conflict. I don't see any difference for the `Theme.palette.directColor1` and now it also works for other colors like `danger1`